galvasr2/align/spark/schemas.py

```diff
@@ -45,7 +45,7 @@
             StructField("aspect_ratio", StringType(), True),
             StructField("backup_location", StringType(), True),
             StructField("closed_captioning", StringType(), True),
-            StructField("collection", ArrayType(StringType(), True), True),
+            StructField("collection", StringType(), True),
```

Comment:
This is excellent. Good find. I've independently checked it on my own. It actually may have caused a loss of data in the forced alignment workflow, since fields other than licenseurl were being incorrectly set to null!